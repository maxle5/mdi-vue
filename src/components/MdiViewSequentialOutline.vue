
<template>
  <svg :width="size" :height="size" :viewBox="viewBox">
      <path d="M3 5V19H21V5H3M19 7V9H5V7H19M19 11V13H5V11H19M5 17V15H19V17H5Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiViewSequentialOutline",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
      computed: {
        viewBox() {
          return `0 0 ${this.size} ${this.size}`;
        }
      }
  })
</script>
