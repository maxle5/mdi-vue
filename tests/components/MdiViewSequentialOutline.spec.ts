
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiViewSequentialOutline from "../../src/components/MdiViewSequentialOutline.vue";

test("MdiViewSequentialOutline snapshot", () => {
  const wrapper = mount(MdiViewSequentialOutline, {});
  expect(wrapper.html()).toMatchSnapshot();
});
