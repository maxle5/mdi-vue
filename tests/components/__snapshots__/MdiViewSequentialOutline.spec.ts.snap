// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiViewSequentialOutline snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3 5V19H21V5H3M19 7V9H5V7H19M19 11V13H5V11H19M5 17V15H19V17H5Z" style="fill: currentColor;"></path>
</svg>"
`;
